ehr_analysis.py

```diff
@@ -1,54 +1,59 @@
 from datetime import datetime
-from typing import Union
 from typing import List
+from ehr_part3 import Patient, Lab
 
 """Data Parsing"""
 
-def parse_patient(patient: str) -> List[List[str]]: 
-    with open(patient, "r") as file: 
-        rows = file.readlines() 
-        lists = [] 
-    for row in rows: 
-        row = row.strip() 
-        row = row.split("\t") 
+
+def parse_patient(patfilename: str) -> List[Patient]:
```

Comment:
`mypy` will not like this. Please use `mypy` to validate the type hints. This is the function signature that we want, _and_ we want the function to _actually do this_. Right now it's returning a `list[list[str]]`, but we need to convert each `list[str]` into a `Patient`.

Reply:
this is solved